packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss

```diff
@@ -715,6 +716,7 @@
                             rgba(0, 147, 147, 0.48) 100%
                         )
                         #151717;
+                    border-radius: 0.8rem;
```

Comment:
Border radius is added to every currency and for both light and dark. Isit possible to just add the rule at the top so the child will inherit it. Or create a new classname and assign `border-radius: 0.8rem`.